ui/storybook/src/decorators/WithTemplateVariables.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 The Perses Authors
```

Comment:
This is mostly a move, but I changed enough that git read it as a delete and add.